sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -1101,19 +1068,80 @@ case class UnsetTableProperties(
 
 trait AlterTableCommand extends UnaryCommand {
```

Comment:
I think this is `AlterTableColumnCommand`? For example, ALTER TABLE ... SET TBLPROPERTY does not use it.

Reply:
Updated. Thanks.